Formula/archi-steam-farm.rb

```diff
@@ -1,30 +1,32 @@
 class ArchiSteamFarm < Formula
-  desc "ASF is a C# application that allows you to farm steam cards"
+  desc "A C# (mono) application that allows you to farm steam cards using multiple steam accounts simultaneously."
   homepage "https://github.com/JustArchi/ArchiSteamFarm"
-  url "https://github.com/JustArchi/ArchiSteamFarm/releases/download/2.3.2.0/ASF.zip"
-  sha256 "1a9f50c3cf2eb00e5148bc21a209b0c7c275b6c36c8cae8b4d9b2469bee7ff33"
+  url "https://github.com/JustArchi/ArchiSteamFarm/releases/download/3.0.5.5/ASF-osx-x64.zip"
```

Comment:
Formulae must build everything from source. So either we'll need to use the generic version with `dotnet` or this will need to be moved to Cask. That's why it's not been updated.

Reply:
The previous Formula downloaded the pre-compiled version. I was trying to minimize the changes from the previous version, but I will adjust. Thanks for the help @ilovezfs

Reply:
Right, precompiled version that runs on tap of mono. (We allow prebuilt Java jars, too.) That is allowed. But a prebuilt native Mach-O is not.

That's what I meant about the generic version. You can use the precompiled generic version, but we'd need a `dotnet` formula which doesn't exist yet.

Reply:
That was attempted once before: https://github.com/Homebrew/homebrew-core/pull/3691